front/app/containers/Authentication/useSteps/stepConfig/lightFlow.ts

```diff
@@ -58,22 +52,11 @@ export const lightFlow = (
         }
       },
       CONTINUE_WITH_SSO: (ssoProvider: SSOProviderWithoutVienna) => {
-        switch (ssoProvider) {
-          case 'google':
-            setCurrentStep('light-flow:google-policies');
-            break;
-          case 'facebook':
-            setCurrentStep('light-flow:facebook-policies');
-            break;
-          case 'azureactivedirectory':
-            setCurrentStep('light-flow:azure-ad-policies');
-            break;
-          case 'azureactivedirectory_b2c':
-            setCurrentStep('light-flow:azure-ad-b2c-policies');
-            break;
-          case 'franceconnect':
-            setCurrentStep('light-flow:france-connect-login');
-            break;
+        if (ssoProvider === 'franceconnect') {
```

Comment:
I guess we have this because france connect is different in some way?

Reply:
Yeah they have a special type of interface that is required by law. See `front/app/components/UI/FranceConnectButton/index.tsx`